Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.cpp

```diff
@@ -26,33 +26,33 @@
 #include "config.h"
 #include "WebPermissionController.h"
 
-
 #include "WebPage.h"
+#include "WebPageProxyIdentifier.h"
```

Comment:
No need to include the same header in .h and .cpp

Reply:
Fixed!